### DISCUSSION TITLE
Mongoose v4 is connecting to MongoDB v5.x

### DISCUSSION THREAD

**Discussion by aayushbisen** [Automattic/mongoose#13388] [Q&A]
We have a project that is using Mongoose v4  ( ^4.13.11 )

We have recently upgraded the MongoDB server to version 5, when doing the deployment the project in production it is not throwing any errors during mongoose connection.

But in the version compatibility [docs](https://mongoosejs.com/docs/compatibility.html) I noticed that only mongoose 6.x and 7.x is compatible with MongoDB 5.x. 

Is there any mistake in the compatibility table or am I reading it wrongly, since the database connection is not failing with mongoose 4.x ?

**Comment by vkarpov15** [comment]
The compatibility table is not wrong. Mongoose 4.x may be able to connect to MongoDB 5, but you won't be able to access some features of MongoDB 5. Further, we don't test Mongoose 4 against MongoDB 5, so there may be bugs we're not aware of. Also, Mongoose 4 is no longer supported, so even if you do find bugs related to MongoDB 5 support, we won't fix them.
